packages/modal/src/Modal.spec.tsx

```diff
@@ -0,0 +1,77 @@
+import React from 'react';
+import { render, cleanup } from '@testing-library/react';
+import 'jest-dom/extend-expect';
+import Modal from './Modal';
+
+afterAll(cleanup);
+
+describe('packages/Modal', () => {
+  const modalId = 'modal';
+  const modalContent = 'Modal Content';
+
+  describe('when rendered with content', () => {
+    const { getByText, getByTestId } = render(
+      <Modal active data-testid={modalId}>
+        <h4>{modalContent}</h4>
+      </Modal>,
+    );
+
+    test('it appears on DOM when active prop is set', () => {
+      const modal = getByTestId(modalId);
+      expect(modal).toBeVisible();
+    });
+
+    test('it renders the children as expected', () => {
+      const content = getByText(modalContent);
+      expect(content).toBeInTheDocument();
+    });
+  });
+
+  describe('when rendered with active equal to false', () => {
+    const { container } = render(
+      <Modal active={false} data-testid={modalId}>
+        <h4>{modalContent}</h4>
+      </Modal>,
+    );
+
+    test('it does not display the modal', () => {
+      expect(container.innerHTML).toBe('');
+    });
+  });
+
+  describe('when modalShouldClose prop is supplied', () => {
+    test('when modalShouldClose returns false', () => {
+      const { getByTestId } = render(
+        <Modal
+          active={true}
+          data-testid="falseModalShouldClose"
+          modalShouldClose={() => false}
+        >
+          <h4>test content</h4>
+        </Modal>,
+      );
+
+      const modal = getByTestId('falseModalShouldClose');
+      window.dispatchEvent(new MouseEvent('click', { bubbles: true }));
+
+      expect(modal).toBeVisible();
+    });
+
+    test('when modalShouldClose returns true', () => {
+      const { getByTestId } = render(
+        <Modal
+          active={true}
+          data-testid="trueModalShouldClose"
+          modalShouldClose={() => true}
+        >
+          <h4>test content</h4>
+        </Modal>,
+      );
+
+      const modal = getByTestId('trueModalShouldClose');
+      window.dispatchEvent(new MouseEvent('click', { bubbles: true }));
+
+      !expect(modal).toBeVisible();
```

Comment:
Is there something like `expect(modal).toNotBeVisible();`? I don't like the pattern of `!expect()`.

Reply:
there is not unfortunately

Reply:
Can't you do `expect(modal).not.toBeVisible()`? It looks like the [examples for `jest-dom`](https://github.com/testing-library/jest-dom) follow that pattern

Reply:
Bump

Reply:
👍